torch/autograd/__init__.py

```diff
@@ -203,6 +207,18 @@ def grad(
         allow_unused (bool, optional): If ``False``, specifying inputs that were not
             used when computing outputs (and therefore their grad is always zero)
             is an error. Defaults to ``False``.
+        batched_grads (bool, optional): If ``True``, the first dimension of each
```

Comment:
For boolean args, I prefer `is_grads_batched` to make it clear from the name that it is boolean and does not contain actual gradients